construct-sturdy-uis-with-xstate/README.md

```diff
@@ -0,0 +1,35 @@
+<h1 align="center"><a href="https://egghead.io/courses/construct-sturdy-uis-with-xstate">Construct Sturdy UIs with XState</a></h1>
+
+<p align="center"><img src="https://d2eip9sf3oo6c2.cloudfront.net/series/square_covers/000/000/402/full/State_Machine.png" width="200"></p>
+
+### About 🔮
+This repo contains notes from Ian Mann's Egghead course [Construct Sturdy UIs with XState](https://egghead.io/courses/construct-sturdy-uis-with-xstate)
+
+These notes contain the same structure as the transcriptions, along with additional rewrites, links to resources, and personal takes on the lesson. Feel free to submit additions to these notes, but please don't remove anything (unless we messed up or misunderstood something).
+
+### Course Description 🔎
+Our applications have lots of states that they can be in.
+
+We usually remember “Loading” and “Successful”. If we’re having a particularly good day, we’ll get “Error” handled as well.
+
+But what about other states that our application can be in?
+
+Things like “Haven’t requested yet” or “The request was successful but there isn’t any data for us to show”.
+
+It’s frustrating to think you’re “done” and then an edge case shows up (and this cycle repeats at least one more time).
+
+Fed up with forgetting to handle the edge cases that appear when developing UIs, Isaac Mann started looking for a solution.
+
+The answer? State machines.
+
+In this course, Isaac will teach you how to plan and visualize your application’s states and the transitions between them through the creation of a React & XState powered Star Wars quiz app.
+
```

Comment:
This is a great set of notes! 🙌

I would suggest adding an emoji legend to be explicit about what each emoji is suggesting. These are some great example:

- https://github.com/eggheadio/eggheadio-course-notes/tree/master/build-an-app-with-the-AWS-cloud-development-kit#emoji-legend-
- https://github.com/kentcdodds/bookshelf#helpful-emoji----------

Reply:
I second this. The emojis are great to have but making sure people know what they mean is important.

Reply:
Added! Thanks for the suggestion :)